kodi_addon_checker/check_dependencies.py

```diff
@@ -122,3 +135,19 @@ def _get_ignore_list(branch_name):
 
     else:
         return common_ignore_deps
+
+
+def check_extensions(report: Report, parsed_xml, addon):
```

Comment:
As this function is not supposed to be called from outside of this file, mark it pseudo private.
```suggestion
def _check_extensions(report: Report, parsed_xml, addon):
```